client-app/shared/account/components/account-navigation.vue

```diff
@@ -2,145 +2,101 @@
-        :to="{ name: 'Quotes' }"
-        :text="$t('quotes.navigation.route_name')"
-        class="quotes-icon"
-      />
+            <VcIcon class="flex-none text-primary" name="minus" />
+
+            <router-link
+              :to="{ name: 'ListDetails', params: { listId: list.id } }"
+              class="line-clamp-2 cursor-pointer py-0.5 font-bold text-neutral hover:text-additional-950"
+              active-class="!text-neutral-950"
+            >
+              {{ list.name }}
+            </router-link>
+          </div>
+        </template>
+      </AccountNavigationLink>
     </VcWidget>
 
     <VcWidget v-if="isCorporateMember" :title="$t(`shared.account.navigation.corporate_title`)" size="sm">
       <AccountNavigationLink
-        :to="{ name: 'CompanyInfo' }"
-        :text="$t('shared.account.navigation.links.company_info')"
-        class="company-icon"
-      />
-
-      <AccountNavigationLink
-        :to="{ name: 'CompanyMembers' }"
-        :text="$t('shared.account.navigation.links.company_members')"
-        class="company-members-icon"
+        v-for="link in desktopCorporateMenuItems?.children"
+        :key="link.id"
+        :to="link.route"
+        :text="link.title"
+        :icon="link.icon"
       />
     </VcWidget>
   </div>
 </template>
 
 <script setup lang="ts">
 import { eagerComputed } from "@vueuse/core";
-import { watchEffect } from "vue";
+import { computed, watchEffect } from "vue";
 import { useRoute } from "vue-router";
-import { useModuleSettings } from "@/core/composables/useModuleSettings";
-import { MODULE_ID as QUOTES_MODULE_ID, ENABLED_KEY as QUOTES_ENABLED_KEY } from "@/modules/quotes/constants";
+import { useNavigations } from "@/core/composables";
 import { useUser } from "@/shared/account/composables/useUser";
 import { useUserOrders } from "@/shared/account/composables/useUserOrders";
 import { useUserOrdersFilter } from "@/shared/account/composables/useUserOrdersFilter";
 import { isActive as isPushMessagesActive } from "@/shared/push-messages/composables/usePushMessages";
 import { useWishlists } from "@/shared/wishlists";
 import AccountNavigationLink from "./account-navigation-link.vue";
+import type { ExtendedMenuLinkType } from "@/core/types";
 
 const route = useRoute();
 const { isCorporateMember } = useUser();
 const { lists, fetchWishlists } = useWishlists();
 const { facets } = useUserOrders({});
 const { filterData, applyFilters } = useUserOrdersFilter();
+const { desktopAccountMenuItems, desktopCorporateMenuItems } = useNavigations();
 
 const isListDetails = eagerComputed(() => route.name === "ListDetails");
 const isOrdersPage = eagerComputed(() => route.name === "Orders");
 
-const { isEnabled: isQuotesEnabled } = useModuleSettings(QUOTES_MODULE_ID);
+function canShowItem(item: ExtendedMenuLinkType) {
+  if (isCorporateMember && item.id === "addresses") {
+    return false;
+  }
+  // eslint-disable-next-line sonarjs/prefer-single-boolean-return
```

Comment:
Make the single Boolean return why not?

Reply:
this looks shorter but less readable as for me, and a bit more complicated for adding and modifying conditions:

```ts
function canShowItem(item: ExtendedMenuLinkType) {
  if (isCorporateMember && item.id === "addresses") {
    return false;
  }

  return isPushMessagesActive && item.id === "notifications";
}
```

comparing to (we clearly see conditions to hide the item otherwise we show it):

```ts
function canShowItem(item: ExtendedMenuLinkType) {
  if (isCorporateMember && item.id === "addresses") {
    return false;
  }
  // eslint-disable-next-line sonarjs/prefer-single-boolean-return
  if (!isPushMessagesActive && item.id === "notifications") {
    return false;
  }

  return true;
}
```

but yeah, maybe that just a matter of taste, and better not to use ` // eslint-disable-next-line`